views/layout.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" type="text/css" href="/style.css">
+</head>
+<body>
+  <div class="button">
+      <input type="button" onclick="location.href='/apartments';" value="VIEW APARTMENTS" />
+      <input type="button" onclick="location.href='/apartments/new';" value="ADD APARTMENT" />
+      <input type="button" onclick="location.href='/tenants';" value="VIEW TENANTS" />
+      <input type="button" onclick="location.href='/tenants/new';" value="ADD TENANT" />
+    </div>
+  <div class="content">
+      <%= yield %>
```

Comment:
Sinatra places all of the other views inside of this <%= yield %>. This means that you no longer need to have the <!DOCTYPE html> <html> yada yada stuff in any of the other views